src/components/contact_page.js

```diff
@@ -0,0 +1,77 @@
+/* importing React Combination
+importing material ui Grid layout to adjust screen
+importing material-ui TextFeild to add input box*/
+import React, { Component } from 'react'
+import Grid from '@material-ui/core/Grid'
+import TextField from '@material-ui/core/TextField'
+/* creating a class Contact_page */
+class Contact_Page extends Component {
```

Comment:
change classname to ContactPage - Camel case, no underscore

Reply:
Class name changed to camel case .Please check contact-page .